module-2/backend/week_one.md

```diff
@@ -5,30 +5,125 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. List the five common HTTP verbs and what the purpose is of each verb.
+  POST, GET, PUT, DELETE
+
 2. What is Sinatra?
+  A lightweight version of Ruby on Rails.
+  
 4. What is MVC?
+  Model View Controller
+  
 5. Why do we follow conventions when creating our actions/path names in our Sinatra routes?
+  Consistency throughout the code.  Makes for easier debugging/troubleshooting.
+  
 6. What types of variables are accessible in our view templates without explicitly passing them?
+  Instance, Local, and Class variables.
+
 7. Given the following block of code, how would I pass an instance variable `count` with a value of `1` to my `index.erb` template?
   
   ```ruby
   get '/horses' do
+    @count = 1
     erb :index
   end
   ```
 
 8. In the same code block, how would I pass a local variable `name` with a value of `Mr. Ed`?
+    ```ruby
+  get '/horses' do
+    name = 'Mr. Ed'
+    @count = 1
+    erb :index
+  end
+  ```
+
 9. What's the purpose of ERB?
+  It is utilized by sinatra to allow Ruby code to be passed through CSS & HTML.
+  
 10. Why do I need a development AND test database?
+  Such that you do not contaminate the test assertions by making permanant alterations to the database.
+
 11. What's responsive design?
+  When software looks for the client's display requirements and changes the display accordingly to the user's display needs.
+  
 12. What is CRUD and why is it important?
+  It gives us a framework to build our main controller file.

```

Comment:
CRUD stands for Create, Read, Update, and Delete. These are the four primary actions we can take with our data